src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
I will add a obsolete tag to make it a soft fail. We can then replace it later with whatever comes